workers/tests/unittests/test_ohsome_client.py

```diff
@@ -40,15 +40,31 @@ def setUp(self) -> None:
 
     def test_query_valid_response(self) -> None:
         with patch("httpx.AsyncClient.post", new_callable=AsyncMock) as mock_request:
-            mock_request.return_value = httpx.Response(200, content=self.valid_geojson)
+            mock_request.return_value = httpx.Response(
+                200,
+                content=self.valid_geojson,
+                request=httpx.Request("POST", "mock.org"),
```

Comment:
Actually this will make a request to a website. It just create the object which is needed by the httpx client to make a request: https://www.python-httpx.org/api/#request

I substitude mock.org with "url" to make the generic naming more clear.

Reply:
Changed URL to http://www.example.org/